test/suite/test_tiered19.py

```diff
@@ -42,16 +44,75 @@ class test_tiered19(wttest.WiredTigerTestCase, TieredConfigMixin):
         ('gcp_store', dict(is_tiered = True,
             is_local_storage = False,
             auth_token = get_auth_token('gcp_store'), 
-            bucket = 'pythontest',
+            bucket = 'test_tiered19',
             bucket_prefix = "pfx_",
             ss_name = 'gcp_store')),
     ]
 
     # Make scenarios for different cloud service providers
     scenarios = make_scenarios(tiered_storage_sources)
+
+    def get_storage_source(self):
+        return self.conn.get_storage_source(self.ss_name)
+
+    def get_fs_config(self, prefix = '', cache_dir = ''):
+        conf = ''
+        if prefix:
+            conf += ',prefix=' + prefix
+        if cache_dir:
+            conf += ',cache_directory=' + cache_dir
+        return conf
     
     def conn_extensions(self, extlist):
         TieredConfigMixin.conn_extensions(self, extlist)
 
     def test_gcp_and_azure(self): 
+        # Test some basic functionality of the storage source API, calling
```

Comment:
Are we testing some or all of the functionality? If "some", it would be useful to explain what is and is not covered by the testing.

Reply:
This function should test all of the basic functionalities but since some of the functions haven't been implemented so I used 'some' here. I've deleted it as we will be adding more testings for the rest of the basic functionalities here therefore the 'some' is not needed.